frontend/public/locales/en/application.json

```diff
@@ -57,6 +57,10 @@
     "repo.type.label.chart": "Chart name",
     "repo.type.label.packageFilterVersion": "Package version",
     "repo.type.label.bucketPath": "Subfolder",
+    "resource.channels": "Channels",
+    "resource.placementrules": "Placement rules",
+    "resource.placements": "Placements",
+    "resource.subscriptions": "Subscriptions",
```

Comment:
We can revert this change, right? And if you can confirm if this file is no longer used, let's remove it.

Reply:
Seems like overview is still using it.